lib/index.js

```diff
@@ -6,6 +6,8 @@ const pathParser = require('path-parser');
 const _ = require('lodash');
 const packageJson = require('../package');
 
+let bodyAbbreviationLength = -1;
```

Comment:
Don't love this global.  My thought on where to put it instead was to save it as a property of the logger.  I.e. in `initLogger`, assign the newly created logger to a variable then do `logger.bodyAbbreviationLength = abbreviation` before returning it.  It's still a little squirrely, passing the parameter into where it's used via a custom property rather than a regular function argument, but the latter would probably require threading the argument through a lot of places, so a property seems like a reasonable compromise.

Note also that the initialization to `-1` is not essential here since it's also done by setting it as the default value for the option and always passing the option value to `initLogger`.

Reply:
That's a good idea, it pained me to use a global like that but I couldn't think of how else to handle it at the time.